.changeset/eighty-chefs-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': major
+---
+
+Renamed `machine.withConfig(...)` to `machine.withOptions(...)`. https://github.com/davidkpiano/xstate/projects/1#card-40598347
```

Comment:
This card doesn't seem to contain a lot of information that would be helpful for readers - I think its mention can be safely removed from here.
```suggestion
Renamed `machine.withConfig(...)` to `machine.withOptions(...)`.
```